sdk/maps/maps-render-rest/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 2.0.0-beta.1 (Unreleased)
+
+### Features Added
+
+### Breaking Changes
+
+- The API endpoint for `GetMapStaticImage` has been updated. The `format` parameter has been removed from the path, changing the usage from `(path: "/map/static/{format}", format: "png")` to `(path: "/map/static")`.
```

Comment:
We should also mention `layer` and `style` is replaced with `tilesetId` in the breaking changes, and the new `trafficLayer` support in features added section.

```diff
export interface RenderGetMapStaticImageQueryParamProperties {
    bbox?: Array<number>;
    center?: Array<number>;
    height?: number;
    language?: string;
-   layer?: "basic" | "hybrid" | "labels";
    path?: string;
    pins?: string;
-   style?: "main" | "dark";
+   tilesetId?: "microsoft.base" | "microsoft.base.labels" | "microsoft.base.hybrid" | "microsoft.terra.main" | "microsoft.base.road" | "microsoft.base.darkgrey" | "microsoft.base.labels.road" | "microsoft.base.labels.darkgrey" | "microsoft.base.hybrid.road" | "microsoft.base.hybrid.darkgrey" | "microsoft.imagery" | "microsoft.weather.radar.main" | "microsoft.weather.infrared.main" | "microsoft.traffic.absolute" | "microsoft.traffic.absolute.main" | "microsoft.traffic.relative" | "microsoft.traffic.relative.main" | "microsoft.traffic.relative.dark" | "microsoft.traffic.delay" | "microsoft.traffic.delay.main" | "microsoft.traffic.reduced.main" | "microsoft.traffic.incident";
+   trafficLayer?: "microsoft.traffic.relative.main" | "none";
    view?: "AE" | "AR" | "BH" | "IN" | "IQ" | "JO" | "KW" | "LB" | "MA" | "OM" | "PK" | "PS" | "QA" | "SA" | "SY" | "YE" | "Auto" | "Unified";
    width?: number;
    zoom?: number;	
}
```




































And in the feature added, we should mention the new tiles support `trafficLayer`